### PR TITLE
Replace 'past speakers' with 'current speakers' section

### LINKED ISSUE #292: Replace 'Past Speakers' section with 'Speakers' section
- Replace the content in the section with the speakers of this year instead
- Replace the title 'Past Speakers' with 'Speakers'
- control the content by showing one section or the other with a feature flag.

### LINKED ISSUE #326: Add Moderator section
We should add a section under "Speakers" for the moderator. It can look exactly the same as the speakers section:

Title: "Moderator"
Data: Sara Vieira, https://iamsaravieira.com, https://twitter.com/NikkitaFTW

Can you create a PR?

### LINKED ISSUE #330: Use regular Title instead of pill title for 'Speakers' and 'Moderator' section
* as we changed the UI of the homepage by adding a new section I think it would be more logical using H1 for the 'Speakers' header fro the Speakers section

Screenshot 2023-05-19 at 10 18 13

_Originally posted by @oscard0m in https://github.com/mainmatter/eurorust.eu/issues/308#issuecomment-1558859734_

### PR DESCRIPTION
**Pull request by oscard0m**
# Description
- Change title from **Past Speakers** to **Speakers**
- Remove old speakers
- Add speakers
   - Niko Matsakis
   - ~Ashley Williams~ TBA
   - ~Iryna Shestak~ TBA
   - ~Lisa Passing~ TBA
- Add 'Moderator' section
   - Sara Vieira  

## Out of scope
Changing how the title of the section looks like will be addressed in https://github.com/mainmatter/eurorust.eu/issues/330

# Context
Fixes #292
Fixes #326
Fixes #330

### CODE REVIEW

**Comment by oscard0m** [comment]
Once re-approved, let me know if we can go live.

**Comment by oscard0m** [comment]
> doesn't it look strange to have one person in the middle of a huge empty screen? I understand this seemed to be the easiest way to just duplicate the Speakers section, but I would never come up with a UI solution like this.
> 
> what do you think?
> 
> Screenshot 2023-05-24 at 15 02 38

@davidhajba Do you mean the space between elements is too much or do you mean we should re-think the design for this in Sketch?

**Comment by davidhajba** [comment]
things start to look very different than the [ORIGINAL UI](https://www.sketch.com/s/635e7434-03d6-46c0-8dad-fa9f7fce9021/a/VrZgkDr)

I think we have 3 options:
* Add everything quickly without any conscious UI planning 
* Collect all the new and possible future elements and rethink the UI - so I can be responsible for the overall look
* Add everything quickly now & meanwhile Collect all the new and possible future elements and rethink the UI

@marcoow please let me know how to proceed

**Comment by marcoow** [comment]
We'll add this as it is for now. We can rethink things but can't redo the entire layout – we can't invest huge amounts of time in these conference websites

**Comment by davidhajba** [comment]
`We can rethink things but can't redo the entire layout – we can't invest huge amounts of time in these conference websites`
thank you - I understand

File: content/_index.md [hunk 1]
```diff
@@ -56,39 +56,33 @@
...
-	image="aissata-maiga"
-	twitter="ai_maiga"
-	web="https://wronganswer.blog/"
-	hide_link=true
+	name="Armin Ronacher"
+	company="Sentry"
+	job_title="Principal Architect"
+	image="armin-ronacher.webp"
+	twitter="mitsuhiko"
+	web="https://lucumr.pocoo.org/"
```
Comment (marcoow) [inline content/_index.md:80]:
```suggestion
	name="Iryna Shestak"
	company="Apollo GraphQL"
	job_title="Senior Staff Rust Engineer"
	image="armin-ronacher.webp"
	twitter="_lrlna"
	web="https://toot.cafe/@lrlna"
```

We're not announcing Armin (for now, maybe later). We forgot Iryna here anyway though 🤦

**Review by marcoow — CHANGES_REQUESTED** [review]

**Review by marcoow — APPROVED** [review]

**Comment by oscard0m** [comment]
@marcoow Do I merge? 🚀

**Comment by marcoow** [comment]
no, Sarah and Emma will let you know next week

**Comment by marcoow** [comment]
Sara isn't yet announced as the moderator

**Pull request merged**